postgres-inserter/main/main.go

```diff
@@ -1,18 +1,40 @@
 package main
 
 import (
+	"github.com/codeuniversity/smag-mvp/kafka"
 	inserter "github.com/codeuniversity/smag-mvp/postgres-inserter"
 	"github.com/codeuniversity/smag-mvp/service"
 	"github.com/codeuniversity/smag-mvp/utils"
 )
 
 func main() {
+	var i *inserter.Inserter
 
-	kafkaAddress := utils.GetStringFromEnvWithDefault("KAFKA_ADDRESS", "127.0.0.1:9092")
 	postgresHost := utils.GetStringFromEnvWithDefault("POSTGRES_HOST", "127.0.0.1")
 	postgresPassword := utils.GetStringFromEnvWithDefault("POSTGRES_PASSWORD", "")
 
-	i := inserter.New(kafkaAddress, postgresHost, postgresPassword)
+	isUserDiscovery, err := utils.GetBoolFromEnvWithDefault("USER_DISCOVERY", false)
```

Comment:
You could move this inside the `kafka.GetInserterConfig` and return `isUserDiscovery`, since we need to do that for every inserter